modules/features/manatime_feature_timesheets/manatime_feature_timesheets.module

```diff
@@ -7,6 +8,22 @@
 include_once 'manatime_feature_timesheets.features.inc';
 
 /**
+ * Implements hook_menu().
+ */
+function manatime_feature_timesheets_menu() {
+  $items = array();
+
+  $items['timesheet/add-new-form/%/%ctools_js'] = array(
+    'page callback' => 'manatime_feature_timesheets_add_row_callback',
+    'page arguments' => array(2, 3),
+    'access callback' => 'user_is_logged_in',
```

Comment:
I know I set this access callback, but I think we need to modify it somehow so that we check user is editing HIS/HER timesheet. Maybe create an access callback, pass the node id as access arguments and verify if user is owner of that timesheet.